cfpb/consumerfinance.gov
# Convert squashed migrations to regular ones

The Django documentation on squashing migrations includes a section on converting squashed migrations to regular ones:

https://docs.djangoproject.com/en/4.2/topics/migrations/#squashing-migrations

In the past when we've squashed migrations we've deleted the old migration files but neglected to remove the `replaces` attribute or update migrations that relied on old migrations that were replaced.

This commit updates all existing squashed migrations to convert them to regular ones, in preparation for a future squashing (Note: because you can't re-squash migrations that are already squashed).

## How to test this PR

To test this change, you can confirm that you can `refresh-data.sh` either with a production dump or with test.sql.gz, and then run `cfgov/manage.py makemigrations --dry-run`, and no changes will be detected.

Alternatively you can create a brand new empty database by following the commands documented at:

https://cfpb.github.io/consumerfinance.gov/installation/#postgresql-and-elasticsearch-from-homebrew

and then running `./initial-data.sh` to create an empty site.

## Checklist
